### PR TITLE
DS-2618: Process mail.server.disabled in test-email

### PR DESCRIPTION
**Pull request by rradillen**
https://jira.duraspace.org/browse/DS-2618

### CODE REVIEW

**Comment by KevinVdV** [comment]
+1

File: dspace-api/src/main/java/org/dspace/core/Email.java [hunk 1]
```diff
@@ -7,38 +7,21 @@
...
-import javax.mail.Transport;
+import javax.mail.*;
 import javax.mail.internet.InternetAddress;
 import javax.mail.internet.MimeBodyPart;
 import javax.mail.internet.MimeMessage;
 import javax.mail.internet.MimeMultipart;
-import org.apache.log4j.Logger;
-import org.dspace.services.EmailService;
-import org.dspace.utils.DSpace;
+import java.io.*;
```
Comment (helix84) [inline dspace-api/src/main/java/org/dspace/core/Email.java]: I understand star imports may be a sort of a religious issue with percieved pros and cons on both sides, but this is a bug fix, so let's not touch these.

Comment (rradillen) [inline dspace-api/src/main/java/org/dspace/core/Email.java]: right that only confuses the issue. Removing

**Comment by helix84** [comment]
+1

**Pull request merged**